example/openstack_provision_ex.json

```diff
@@ -7,5 +7,6 @@
     "flavor": "<flavor>",
     "key_pair": "<key_pair>",
     "vm_name": "<instance_name>",
-    "fip_pool": "<floating_ip_pool>"
+    "fip_pool": "<floating_ip_pool>",
+    "floating_ip_id": "<floating_ip_id>"
```

Comment:
maybe it is too late to change, but I personally would like to see consistency in the floating ip name - something like - if we used `fip_pool` why wouldn't we use `fip_id`, or vice versa? Not necessary to change it here though. We may add a new issue to review naming convention for variables.

Reply:
Yes we can open an issue for this.  I guess I never really gave it enough thought.  I believe the discrepancy came from Steve creating the automation request input, where he created fip_pool, and I created the OpenStack vm creation w/ floating_ip_id (the key used in Cloudforms is floating_ip_address, but I changed our input name to id, just to make it clear on what it was, but I wasn't thinking about consistency with the rest of the input just trying to be clear and close to the Cloudforms key name).

Reply:
get_floating_ip requires the CF/MIQ cloud_network_id and cloud_tenant_id. The cloud network id needs to be the one the correlates to the external network. This does relate to the fip_pool but do not know where the name came from. But agree the names should correlate.